Wrapping/Generators/CastXML/CMakeLists.txt

```diff
@@ -171,15 +171,6 @@ mark_as_advanced(CASTXML_EXECUTABLE)
 set(ITK_WRAP_CASTXML_SOURCE_DIR "${CMAKE_CURRENT_SOURCE_DIR}" CACHE INTERNAL "castxml source dir")
 
 
-macro(itk_wrap_include_castxml include_file)
```

Comment:
Perhaps either squash all `inline` commits together, or separate this one into two commits.